.github/workflows/docker.yml

```diff
@@ -0,0 +1,22 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - "main"
+  workflow_dispatch:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Push to GitHub Packages
+        uses: docker/build-push-action@v1
+        with:
+          push: true
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          registry: ghcr.io
+          repository: ${{ github.repository }}/runtime
```

Comment:
This will give us the following the image `flatpak-platform/runtime` & then it can be used as such ` ghcr.io/elementary/flatpak-platform:daily`